crates/red_knot_python_semantic/resources/mdtest/expression/boolean.md

```diff
@@ -55,7 +55,7 @@ reveal_type("x" or "y" and "")  # revealed: Literal["x"]
 ## Evaluates to builtin
 
 ```py path=a.py
-redefined_builtin_bool = bool
+redefined_builtin_bool: type[bool] = bool
```

Comment:
This whole test feels somewhat contrived, but it's maybe a symptom of a larger problem in which we don't understand re-exports of symbols that are not also re*declared*.

Reply:
It's not that we don't understand them, it's that we now accurately reflect the fact that they could be reassigned to something else.

We do consider imports to be declarations, so the common case of re-exporting (that uses an import, not an assignment) wouldn't be subject to this.